### PR TITLE
Update License 

### PR DESCRIPTION
**Pull request by madhikarma**
# Description
- Updated license file name to remove markdown
# Status
- [x] Review

### CODE REVIEW

**Comment by danieladias** [comment]
I know it's not related but do you mind adding the code of conduct file to this repo as well with a mention in the readme file?

**Comment by madhikarma** [comment]
Sure @danieladias I've added that missing file now

**Comment by danieladias** [comment]
Sorry about this but do you mind adding this in readme:

```
## Contributing

Please note that this project is released with a Contributor Code of Conduct. By participating in this project you agree to abide by its terms. See the [Code of Conduct](CODE_OF_CONDUCT.md) file.
```

File: README.md [hunk 1]
```diff
@@ -56,10 +56,14 @@ fruits.append(apple)
...
 * [Shagun Madhikarmi](mailto:shagun@ustwo.com)
 
+## Contact
+
+[open.source@ustwo.com](mailto:open.source@ustwo.com)
+
 ## License
 
-Array+Remove is released under the MIT License. See the LICENSE file.
+URLSession+Cancellation is released under the MIT License. See [License](LICENSE).
```
Comment (danieladias) [inline README.md]: The library name is wrong

**Review by danieladias — CHANGES_REQUESTED** [review]

**Review by danieladias — APPROVED** [review]

**Pull request merged**